setup.py

```diff
@@ -38,12 +41,51 @@
 import shutil
 from setuptools import setup, find_packages, Extension
 from distutils.core import run_setup
-from Cython.Distutils import build_ext as _build_ext
 
 def compile_enet():
     previousDir = os.getcwd()
     os.chdir("enet")
-    subprocess.Popen(["./prebuild.sh"]).communicate()
+
+    ###
+    ### Prebuild.py start
+    ###
+    import tarfile
+    try:
+      import urllib as urllib_request
+    except ImportError:
+      import urllib.request as urllib_request
+
+    lib_version = "1.3.13"
+    enet_dir = "enet-%s" % lib_version
+    enet_file = "%s.tar.gz" % enet_dir
+    enet_url = "http://enet.bespin.org/download/%s" % enet_file
+
+    if os.path.isfile("pyenet/enet-pyspades.pyx"):
+        os.remove("pyenet/enet-pyspades.pyx")
+    if os.path.isfile("pyenet/enet.so"):
+        os.remove("pyenet/enet.so")
+    if os.path.isdir("pyenet/enet"):
+        shutil.rmtree("pyenet/enet")
+
+    shutil.copyfile("pyenet/enet.pyx", "pyenet/enet-pyspades.pyx")
+    subprocess.Popen(['patch', '-p1', 'pyenet/enet-pyspades.pyx', 'pyspades-pyenet.patch']).communicate()
+
+    if not os.path.isfile(enet_file):
+        print("Downloading enet")
+        urllib_request.urlretrieve(enet_url, enet_file)
```

Comment:
also, we should note that this means that there is no longer a progress bar for the download. I don't think it's a huge problem.